setup.cfg

```diff
@@ -43,9 +43,9 @@ install_requires =
     scikit-image>=0.17.2
     scipy>=1.6.0,<1.10.0
     spherical-geometry>=1.2.22
-    stcal>=1.3.4,<1.4
-    stdatamodels>=1.1.0,<1.2.0
-    stpipe>=0.4.5,<1.0
+    stcal>=1.3.4
+    stdatamodels>=1.1.0
```

Comment:
Now that https://github.com/spacetelescope/stdatamodels/pull/127 has been merged, we should push the minimum for stdatamodels up to 1.2.0, so that jwst gets the updates from that PR.